src/main/java/seedu/address/model/Statistics.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.model;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import javafx.scene.chart.XYChart.Data;
+import seedu.address.model.booking.Booking;
+
+/**
+ * Manages the statistics of the bookings
+ */
+public class Statistics {
+
+    public static final String DATE_PATTERN = "dd MMM yyyy";
+    public static final int MAX_BARS = 20;
+    public static final int MAX_BAR_SIZE = 500;
+
+    public static int getMaxDays() {
+        return MAX_BARS * MAX_BAR_SIZE;
+    }
+
+    private static int getDaysDifference(LocalDate start, LocalDate end) {
+        return Period.between(start, end).getDays();
+    }
+
+    /**
+     * Formats the date into a String.
+     * @param date the date
+     * @return the formatted String
+     */
+    private static String formatDate(LocalDate date) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_PATTERN);
+        return date.format(formatter);
+    }
+
+    /**
+     * Formats 2 dates into a String.
+     * @param start the starting date
+     * @param end the ending date
+     * @return the formatted String
+     */
+    private static String formatDate(LocalDate start, LocalDate end) {
```

Comment:
Do you require start occurs before end?
If so, state it as a pre-condition of the method.